cgmes/cgmes-conversion/src/main/java/com/powsybl/cgmes/conversion/elements/NodeConversion.java

```diff
@@ -163,15 +164,13 @@ public void setVoltageAngleNodeBreaker() {
         VoltageLevel.NodeBreakerView topo = vl.getNodeBreakerView();
         String connectivityNode = id;
         int iidmNode = context.nodeMapping().iidmNodeForConnectivityNode(connectivityNode, vl);
-        if (!topo.hasAttachedEquipment(iidmNode)) {
-            LOG.error("ConnectivityNode {} with voltage and angle is not valid in IIDM", connectivityNode);
-            return;
-        }
         // To obtain a bus for which we want to set voltage:
         // If there no Terminal at this IIDM node,
         // then find from it the first connected node with a Terminal
-        Terminal t = topo.getOptionalTerminal(iidmNode)
-                .orElseGet(() -> new FirstTerminalTraverser(topo, iidmNode).firstTerminal());
+        Terminal t = topo.getTerminal(iidmNode);
+        if (t == null) {
+            t = new FirstTerminalTraverser(topo, iidmNode).firstTerminal();
+        }
```

Comment:
I don't understand why you change this.

Reply:
@lcosson: any feedback?

Reply:
Very sorry about that, it looks like I miss-merged this file. Thank God for PR's!
Correcting...